src/components/Music/Song.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  FaPlay, FaPause, FaStar, FaRegStar,
+} from 'react-icons/fa';
+import '../../styles/Song.css';
+
+export default function Song() {
+  const music = useSelector((state) => state.music);
+  const audio = new Audio(music.music.preview);
+  const [clicked, setClicked] = useState(false);
+
+  const play = () => {
+    audio.play();
+  };
+
+  const pause = () => {
+    audio.pause();
+  };
+
+  const saveSong = () => {
+    if (clicked) {
+      localStorage.removeItem(music.music.title);
+      setClicked(false);
+    } else {
+      localStorage.setItem(music.music.title, JSON.stringify(music.music));
+      setClicked(true);
+    }
+  };
+
+  useEffect(() => {
+    play();
```

Comment:
- [ ] It will be great to ensure that no errors are seen in your application. Although the application is running, errors are popping up. I think these errors are caused by calling the play function inside the`useEffect`. I think the play function should only be called when you want to play the audio. That is when the user clicks on the music item or the play icon.
![Screenshot 2023-05-25 at 13 02 56](https://github.com/Ouail-01/Musify_React-Capstone/assets/32361963/8ab6a23c-6d8c-4f20-9794-d360e5e9cc43)